cypress/integration/Selection.spec.js

```diff
@@ -0,0 +1,14 @@
+describe('Selection page and navigation to it', () => {
+    beforeEach(() => {
+      cy.visit('http://localhost:3000')
+
+    })
+
+    it('should navigate to a selection screen after clicking on the zip code input, typing 5 numbers and hitting enter', () => {
```

Comment:
Good starting test, we'll add on to this as elements and styling become more concrete